pkg/raft/Makefile

```diff
@@ -0,0 +1,28 @@
+GO_TEST_FLAGS?=
```

Comment:
What is the plan for this Makefile? Preserve it? Strip it out and replace with Bazel?

Similar question about the scripts that it calls.

Reply:
Removed it. This package is now streamlined into our CI.